.github/workflows/definitelyTyped.yml

```diff
@@ -0,0 +1,55 @@
+name: DefinitelyTyped Tests
+
+on:
+  push:
+    branches:
+      - 'feature/definitelytype'
+    paths:
+      - 'src/**'
+      - 'definitelyTyped/**'
+
+jobs:
+  performance:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: install ts auto mock
+        run: |
+          npm install
+          npm run build
+        env:
+          CI: true
+      - name: definitelyTyped
+        run: |
+          cd definitelyTyped
+          npm install
+          npm run install-types
+          npm run process-types TYPES=2
+      - name: create environment variable date
+        run: echo "::set-env name=DATE::$(date +%Y-%m-%dT%H-%M-%S)"
+      - name: definitelyTyped push changes
+        run: |
+          git config --global user.name 'typescripttdd'
+          git config --global user.email 'typescripttdd@gmail.com'
+      - name: Create commits
+        run: |
+          git add data/*
+          git checkout .
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2.0.0
+        with:
+          commit-message: Add DefinitelyTyped Tests data
+          committer: typescripttdd <typescripttdd@gmail.com>
+          author: typescripttdd <typescripttdd@gmail.com>
+          title: DefinitelyTyped Tests Data on ${{ env.DATE }}
+          branch: definitely-typed-tests-${{ env.DATE }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Can you change this one to: token: ${{ secrets.GITHUB_TOKEN_TYPESCRIPTTDD }}

If you use the token I just mentioned the PR will trigger a new build on master  otherwise it will not.(And on master is required)

you can see this issue https://github.com/peter-evans/create-pull-request/issues/48 to understand a bit more why.

Reply:
done!